fluent-bit/http/values.yaml

```diff
@@ -174,24 +174,26 @@ fluent-bit:
           Header                private_key ${PRIVATE_KEY}
           compress              gzip
           Retry_Limit           False
+          net.keepalive         off
```

Comment:
why do we want to add it?

Reply:
some customers get the error : [error] [http_client] broken connection to api.coralogix.com:443 
because we don't support using the same connection. this option make sure that fluentbit wont use the same connection.